README.md

```diff
@@ -1,4 +1,4 @@
-# README - Georgia Tech Smoothing and Mapping Library
+# Georgia Tech Smoothing and Mapping Library
```

Comment:
Can you please leave this as saying `README - Georgia Tech Smoothing and Mapping Library`?

Reply:
Ok!